spec/api_config_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'spec_helper'
+
+describe Quandl::ApiConfig do
+  context 'threading' do
+    context '.api_key' do
+      let(:threads) { [] }
+      let(:default_key) { (0...8).map { (65 + rand(26)).chr }.join }
+
+      it 'should allows for a default value' do
+        Quandl::ApiConfig.api_key = default_key
```

Comment:
so from what I understand, if we specify an ApiConfig in the main thread, The `Quandl::ApiConfig.api_key` value cannot be changed a `Thread.new` ?

Can it be changed in the main thread?